browserstack.err

```diff
@@ -0,0 +1 @@
+[object Object]
```

Comment:
Can this file and the `local.log` be removed from the PR?  I don't think they need to be in the project.

Reply:
Thanks for catching that!